doc/contributing.rst

```diff
@@ -0,0 +1,208 @@
+=============
+Contributing
+=============
+
+pyxem is meant to be a community maintained project. We welcome
+contributions in the form of bug reports, documentation, code, feature requests,
+and more. These guidelines provide resources on how best to contribute.
+
+For new users, checking out the `GitHub guides <https://guides.github.com>`_ are
+recommended.
+
+Questions, comments, and feedback
+=================================
+
+Have any questions, comments, suggestions for improvements, or any other
+inquiries regarding the project? Feel free to
+`ask a question <https://github.com/pyxem/pyxem/discussions>`_,
+`open an issue <https://github.com/pyxem/pyxem/issues>`_ or
+`make a pull request <https://github.com/pyxem/pyxem/pulls>`_ in our GitHub
+repository.
+
+Code of Conduct
+===============
+
+pyxem has a Code of conduct that should be honoured
+by everyone who participates in the pyxem community.
+
+.. _setting-up-a-development-installation:
+
+Setting up a development installation
```

Comment:
I'm starting to think that this contributing guide, which I wrote for kikuchipy based on others guides (scikit-image, HyperSpy), can be a bit intimidating. Adding some minimal contributing guide line before this might be preferable, so that a first contributor with very limited `git` experience isn't intimidated from contributing. I think we'll add such a note for kikuchipy.